lib/soapy_cake/response.rb

```diff
@@ -22,6 +22,14 @@ def to_enum
       end
     end
 
+    def to_xml
```

Comment:
Why? We already have the xml as a single big string. Can't it just simply return `body.to_s`?